### DISCUSSION TITLE
Please help me find my account back

### DISCUSSION THREAD

**Discussion by tokyojava** [community/community#145218] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hi there, I used to own `https://github.com/sakopqiu` via this account `sakopqiu@gmail.com`, and I have some important private project attached to this account.

At the moment, even if I login using `sakopqiu@gmail.com`, it takes me to another github I own, which is `https://github.com/tokyojava`,

I am lost, can some one help me?

**Comment by Rawknee-69** [comment]
Here is the answer to your problem , do mark it as an answer if you gys found it useful .... Marking this as answer found help the others with the same problem . Hope you understand , here is the solution to your problem below .
@tokyojava 


![help](https://github.com/user-attachments/assets/b90d3213-98a0-41fb-a30e-4e003654816a)


1.Recover Original Account (sakopqiu) : Visit the [GitHub Account Recovery Page](https://github.com/login/forgot_password).
Enter sakopqiu@gmail.com to receive a password reset link.
If sakopqiu exists and is linked to this email, you should be able to reset the password and log in.

2. Check for Multiple Emails in Accounts : After logging into each account (sakopqiu and tokyojava), check for secondary email addresses in [Email Settings](https://github.com/settings/emails).
GitHub allows multiple emails for a single account, which could lead to confusion.

and this will help to find the account linked with the same email

**Comment by mecodeatlas** [comment]
Hi @tokyojava  👋🏾,It looks like @Rawknee-69's response was the answer to your question here. Can you please mark their response as the answer? When you mark a question as an answer, GitHub will highlight the comment and replies to the comment to help visitors quickly find the answer.
![image](https://github.com/user-attachments/assets/b89e7161-c2d1-47e2-a058-242df633e6b2)
